### ISSUE TITLE
The block will be reset when the player doesn't have a bed set

### ISSUE THREAD

**Issue by YTiceice** [MultiPaper/MultiPaper#369]
This is similar to the problem I reported on Discord before. After a long time of testing, it may be triggered like this. When the player tries to use `/home bed` to return to the spawn point they set, if his bed is destroyed and cannot return to the spawn point, The chunks there will be reset to how they were when they were first generated.
Essentials is using this https://github.com/CritterzNFT/Essentials
Here's a video of what I'm trying to reproduce, with just me on the server:
https://youtu.be/fQd262VN9xA

The server version `git-MultiPaper-"bc53f8e" (MC: 1.19.3)*`
Installed plugins `Essentials, spark`

Console log of test1
```
Koyorice_ issued server command: /tp 5126 80 -1918
[MultiPaperConnection] Downloaded synced file plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperFileSyncer] Uploaded plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
Koyorice_ issued server command: /tprandom
[MultiPaperFileSyncer] Uploaded plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperConnection] Downloaded synced file plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
Koyorice_ issued server command: /home bed
Koyorice_ issued server command: /tp 5126 80 -1918
[MultiPaperFileSyncer] Uploaded plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperConnection] Downloaded synced file plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
```
Console log of test2
```
[MultiPaperFileSyncer] Uploaded plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperConnection] Downloaded synced file plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperConnection] Downloaded synced file plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperFileSyncer] Uploaded plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperConnection] Downloaded synced file plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
[MultiPaperFileSyncer] Uploaded plugins/Essentials/userdata/bd15a434-3660-40b4-a17f-324736e2e41b.yml (2KB)
```

**Comment by YTiceice** [comment]
I just tried running with `git-MultiPaper-"8bdca54" (MC: 1.19.4)*`, but the problem still occurs, please feel free to let me know if you need more information!

**Comment by PureGero** [comment]
Fixed by #490ec31 on the 1.20.1 dev branch

**Comment by YTiceice** [comment]
Thank you!
